src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -180,6 +180,24 @@ public void DroppedExportProcessorItems(string exportProcessorName, string expor
             }
         }
 
+        [NonEvent]
+        public void MeterProviderSdkEvent(string message)
```

Comment:
for these ones, we don't need the NonEvent route, we can make it directly call the method (and EventSource natively handles checking if a level is enabled/not enabled)

Reply:
Removed the `NonEvent` method for this.